skype/skype-ps/skype/Get-CsTenantNetworkRegion.md

```diff
@@ -0,0 +1,116 @@
+---
+external help file: Microsoft.Rtc.Management.Hosted.dll-help.xml
+applicable: Skype for Business Online
+title: Get-CsTenantNetworkRegion
+schema: 2.0.0
+---
+
+# Get-CsTenantNetworkRegion
+
+## SYNOPSIS
+
+## SYNTAX
+
+```
+Get-CsTenantNetworkRegion [-LocalStore] [[-Identity] <Object>] [-Tenant <Object>] [-Filter <Object>] [-AsJob]
+```

```

Comment:
There's an quirk with PlatyPS in that it sets all parameters to be Objects, when in fact some are strings or numerical.  I can't remember if this is fixed in a later version.  Ken would know.

Meanwhile, you should double check the cmdlet source code to see if the parameter types are correct, and update as needed.

Reply:
New PR with fix. Need to add -Session param when running platyPS command to avoid this issue